main.go

```diff
@@ -66,6 +66,10 @@ var (
 	log    = ctrl.Log.WithName("setup")
 )
 
+// Namespace for standalone policy users.
+// Policies applied by users are deployed here. Used only in non-hosted mode.
+const ocmPolicyNs = "open-cluster-management-policies"
```

Comment:
Should this additional namespace be enabled by a boolean flag? Though it could be unlikely a user creates an `open-cluster-management-policies` namespace AND puts ConfigurationPolicies in it, but having it as a flag would be a clear tuning point for users. (I'm not sure whether that matters, though.)

Reply:
What should be the default value? enable or disable as a default

Reply:
Another one if the user create the namespace and added the configPolicy. That configpolicy is not a standalone policy?

Reply:
I'd say disabled by default (and then we can enable it in the addon-controller).
It is a standalone policy, but I feel like the standalone should be opt-in since the current use-case is multicluster with ACM and having it enabled could be unexpected if a managed cluster user happens to see that standalone is an option and tries to create ConfigurationPolicies without consulting the SysAdmin first.

Reply:
I agree with Dale. In terms of system administration, it is better to have enable-disable